pymeasure/instruments/keithley/keithley2260B.py

```diff
@@ -0,0 +1,71 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import (
+    # truncated_range,
+    # truncated_discrete_set,
+    strict_discrete_set,
+)
+# from pymeasure.adapters import VISAAdapter
+from .buffer import KeithleyBuffer
+
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class Keithley2260B(Instrument):
+    """ Represents the Keithley 2260B Power Supply (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+
+    For a connection through tcpip, the socket is hardcoded to 2268,
+        example connection string: 'TCPIP::xxx.xxx.xxx.xxx::2268::SOCKET'
+        the read termination is \n
+
+    .. code-block:: python
+
+        source = Keithley2260B("GPIB::1")
+        source.voltage = 1
+        print(source.voltage)
+        print(source.current)
+        print(source.power)
+        print(source.applied)
+
+    """
+
+    enabled = Instrument.control(
+        "OUTPut?", "OUTPut %d",
+        """A boolean property that controls whether the source is enabled, takes
+        values True or False (1 or 0). """,
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True
+    )
+
+    current = Instrument.control(
+        ":SOUR:CURR?", ":SOUR:CURR %g",
+        """ A floating point property that controls the source current
+        in amps. The allowed range is not fixed. """,
+    )
+
+    voltage = Instrument.control(
+        ":SOUR:VOLT?", ":SOUR:VOLT %g",
+        """ A floating point property that controls the source voltage
+        in volts. The allowed range is not fixed. """,
+    )
+
+    power = Instrument.measurement(
+        ":MEAS:POW?",
+        """ Reads the power (in watts) the dc power supply is putting out.  """
+    )
+
+    applied = Instrument.measurement(
+        ":APPly?",
+        """ Reads the applied voltage (volts) and current (amps), returns as list."""
+    )
+
+    def __init__(self, adapter, **kwargs):
+        super().__init__(adapter, "Keithley 2260B DC Power Supply", read_termination="\n", **kwargs)
+
+    def apply(self, voltage, current):
```

Comment:
have you tried making `applied` above a control, and feed it a tuple of (voltage, current) when setting, with set command `"apply %g,%g"`? I.e., `instr.applied = (15, 30)`
I think this could work, then you could drop this method and use the property instead to set. As the interface is basically equivalent, the API is a bit confusing as-is.

Reply:
I was not aware of this possibility, even though, now that I know of it, it seems obvious. I will make that change, and would at the same time suggest to improve pymeasure's documentation of "Adding instruments" to include this capability.

Reply:
Did you test correct operation with your device?
Yeah, I also did not think of that until now. :grin: 
Good idea! Could you add a remark about this possibility in https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#writing-properties (after the Python string format is mentioned)?

Reply:
Yes I did test it with this device, it works perfectly well. 

I have created #410 for the change in the documentation

Reply:
Great! You can just add this change to the present PR, no need to open a separate one!

Reply:
Never mind, I fixed the other PR already.